libcxx/docs/Status/Cxx2cIssues.csv

```diff
@@ -34,7 +34,7 @@
 "`3957 <https://wg21.link/LWG3957>`__","[container.alloc.reqmts] The value category of v should be claimed","Kona November 2023","","",""
 "`3965 <https://wg21.link/LWG3965>`__","Incorrect example in [format.string.escaped] p3 for formatting of combining characters","Kona November 2023","","","|format|"
 "`3970 <https://wg21.link/LWG3970>`__","[mdspan.syn] Missing definition of ``full_extent_t`` and ``full_extent``","Kona November 2023","","",""
-"`3973 <https://wg21.link/LWG3973>`__","Monadic operations should be ADL-proof","Kona November 2023","","",""
+"`3973 <https://wg21.link/LWG3973>`__","Monadic operations should be ADL-proof","Kona November 2023","|Complete|","19.0",""
```

Comment:
Would it be better to say 18.0 (or even 17.0)?

The monadic operations have been ADL-proof in libc++ since they were implemented (https://github.com/llvm/llvm-project/commit/acce2a315945e386a7be6f014ebe90c2a28f38d9), and actually no change except for diagnostic messages is made since the implementation of [LWG3938](https://cplusplus.github.io/LWG/issue3938) (https://github.com/llvm/llvm-project/commit/9f67143be01c97cf80f6e96e7b4c172d6dc0690e).